test/src/unit-ctx.cc

```diff
@@ -103,3 +105,31 @@ TEST_CASE("C++ API: Test context tags", "[cppapi][ctx-tags]") {
     }
   }
 }
+
+TEST_CASE("C++ API: Test REST version endpoint", "[cppapi][rest][version]") {
+  tiledb::test::VFSTestSetup vfs_test_setup;
+  tiledb::Config config;
+  std::string serialization_format = GENERATE("JSON", "CAPNP");
+  config["rest.server_serialization_format"] = serialization_format;
+  config["rest.server_address"] = "http://127.0.0.1:8181";
+
+  // Only run these tests if the rest client has been initialized
+  if (vfs_test_setup.is_rest()) {
+    // Update the config in each section so that we always have a fresh Context.
+    // Otherwise, the JSON test will initialize rest_tiledb_version_ for CAPNP.
+    DYNAMIC_SECTION(
+        "GET request to retrieve REST tiledb version - "
+        << serialization_format) {
+      vfs_test_setup.update_config(config.ptr().get());
+      auto ctx = vfs_test_setup.ctx();
+      REQUIRE(ctx.ptr()->rest_client().get_rest_version() == "2.28.0");
```

Comment:
We can't have `2.28.0` hard-coded in the test, can we? I'd suggest we get them from `tiledb_version.h` since we always use the same version in REST in our CI.